src/test/java/org/cactoos/scalar/RepeatedTest.java

```diff
@@ -61,7 +61,7 @@ void throwsIfZero() {
                     () -> {
                         throw new IllegalStateException("intended to fail");
                     },
-                    true
+                    "discarded"
```

Comment:
@victornoel Why do you change `true` for `discarded` ?

Reply:
@baudoliver7 to better document what happens here, the value is passed here is used nowhere, so it is discarded in practice. I thought it was better than using true, because it could confuse people as to why true is passed here and what would happen if one passes false, or something else :)